unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/ReportFriendRequestToAnalyticsExtensions.cs

```diff
@@ -9,12 +10,26 @@ public static class ReportFriendRequestToAnalyticsExtensions
             string source,
             IFriendsController friendsController, ISocialAnalytics socialAnalytics)
         {
-            FriendRequest request = friendsController.GetAllocatedFriendRequest(friendRequestId);
-            socialAnalytics.SendFriendRequestError(request?.From, request?.To,
+            bool wasFound = friendsController.GetAllocatedFriendRequest(friendRequestId, out FriendRequest request);
+
+            var description = e is FriendshipException fe
+                ? fe.ErrorCode.ToString()
+                : FriendRequestErrorCodes.Unknown.ToString();
+
+            if (!wasFound)
```

Comment:
Just a suggestion: if you moved `friendsController.GetAllocatedFriendRequest(friendRequestId, out FriendRequest request)` here you would not need a separate variable `wasFound`